[Integration][Snyk] Add more origins to Target blueprint (#443)
port-labs/ocean: integrations/snyk/CHANGELOG.md

<!-- towncrier release notes start -->

# Port_Ocean 0.1.31 (2024-03-21)

### Improvements

> - Add three more origins to the Snyk Target blueprint

Reviewer: ```suggestion
- Expand the origins Enum of the Snyk Target blueprint
```
